drafts/25-xxx-macro-expansion.txt

```diff
@@ -241,28 +389,85 @@ EXAMPLE lbe01:
 
 2.1.3 Line breaks and continuations in macro invocations (fixed-form)
 ---------------------------------------------------------------------
+[Straw poll 1, option C -> mark subsection 2.1.3 as "rejected"]
 
 This subsection describes the handling of line continuations within
 function-like macro invocations in fixed source form.
 
-lc01. An invocation of a function-like macro can be broken across
-      multiple lines. An invocation that begins with an open parenthesis
-      '(' continues until the matching closing parenthesis ')' is found
-      (outside of a comment), which may occur on a subsequent source
-      fragment line.
+lc01. [Straw poll 2, option R] An invocation of a function-like macro
+      can be continued across multiple lines. An invocation that begins
+      with an open parenthesis '(' continues until the matching closing
+      parenthesis ')' is found (outside of a comment).
+
+lc01. [Straw poll 2, option O] An invocation of a function-like macro
+      can be broken across multiple lines. An invocation that begins
+      with an open parenthesis '(' continues until the matching closing
+      parenthesis ')' is found (outside of a comment), which may occur
+      on a subsequent source fragment line.
 
 lc03. Comment lines within the argument list of an invocation of a
       function-like macro are discarded (section 2.1.4).
 
-lc07. If character position 6 of a continuation line contains any
-      character other than a blank or zero, then any trailing blank
-      characters of the continued line are removed along with the
-      newline. Otherwise, the newline of the continued line is replaced
-      with a blank character. Characters in positions 1-6 are removed
-      from the continuation line regardless of the character in position
-      6.
+lc07. [Straw poll 2, option R] If character position 6 of a continuation
```

Comment:
Strictly speaking, a line in fixed-form that has a blank or zero in column 6 is NOT a "continuation line".  I know what you're trying to say here, but this is not right.

Reply:
Resolved in 9077b9f